Explorer/Assets/DCL/PluginSystem/Utils/MordorConstants.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public static class MordorConstants
```

Comment:
We have MORDOR in `public partial class AvatarAttachHandlerSystem`, could you unify them?